File: week-04/day-01/sharpie.py
class Sharpie(object):
    def __init__(self, color, width):
        self.color = color
        self.width = width
        self.ink_amount = 100

    def use(self, ink):
        self.ink_amount -= ink

sharpie = Sharpie("red", 100)

sharpie.use(30)
print(sharpie.ink_amount)

File: week-08/day-01/sharpie.js
function Sharpie(color, width){
  this.color = color;
  this.width = width;
  this.inkAmount = 100;
  this.use = function(){
    this.inkAmount -= this.width;
  }
}
var pen = new Sharpie('red', 1);

function sharpieUser(writingTool){
  for(var i = 0; writingTool.inkAmount != 0 ; i++){
    writingTool.use();
  }
}
sharpieUser(pen);
